app.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const {getTopics } = require('./controllers/topics.controller.js')
+
+
+// Set up. 
+const app = express()
+app.use(express.json())
+
+// End points. 
+app.get('/api/topics', getTopics)
+
+
+// Error handlers. 
+app.use((req, res)=>{
+    res.status(404).send({msg: 'Not Found'})
+})
+
+app.use((err, req, res, next)=>{
+    if (err.status && err.msg) {
+        res.status(err.status).send({msg: err.msg})
+    } else next(err)
+})
+
```

Comment:
You should add an error handler that can deal with 500's here (don't need to test it though)

Reply:
Does express cover this? Is it considered good practice to do it yourself anyway?